README.md

```diff
@@ -3,7 +3,6 @@
 ## Rules
 
 <div class="toc">
```

Comment:
Since we are no longer copying this README.md for Bazel.io (as we currently are for the [Packaging rules](https://www.bazel.io/versions/master/docs/be/pkg.html)), we can turn this into a Markdown list.

FYI the main blocker for generating Skydoc documentation for the `rules_*` repos is adding an overview page for the generated documentation, which I am currently working on. Once that is done, then we can generate Skydoc for each of these repos and serve them, similar to how Rust libraries use [generated Rustdoc](http://rust-num.github.io/num/num/index.html) as their project websites.